utils/scapp.h

```diff
@@ -193,6 +193,11 @@ astcEncoderMode(const char* mode) {
                      <tr><td>exhaustive </td> <td>(equivalent to quality = 100) </td></tr>
                  </table>
                  </dd>
+        <dt>--astc_perceptual</dt>
+                 <dd>The codec should optimize perceptual error, instead of direct
```

Comment:
s/optimize perceptual/optimize for perceptual/

Reply:
done in https://github.com/KhronosGroup/KTX-Software/pull/511/commits/f7dc5c23320d9096b782aae1812317a8c62fad07